gstreamer-video/src/video_info.rs

```diff
@@ -166,6 +166,14 @@ impl fmt::Display for ::VideoColorimetry {
 
 pub struct VideoInfo(pub(crate) ffi::GstVideoInfo);
 
+impl fmt::Debug for VideoInfo {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{:?} - ({:?}x{:?} - {:?}bytes)", self.format(),
+            self.width(), self.height(), self.size())
+    }
+}
```

Comment:
Once I regenerate gstreamer-sys soonish, you should be able to simply #[derive(Debug)] (ffi::GstVideoInfo would implement Debug already).